fetch/programmer-humour/script.js

```diff
@@ -0,0 +1,27 @@
+async function fetchLatestXKCD() {
+  const url = 'https://xkcd.now.sh/?comic=latest';
```

Comment:
nit: One way to improve your code is to use a constant for the URL that is defined _outside_ of your function. This will make it easier to maintain and update in the future.